src/blocking.rs

```diff
@@ -273,7 +273,7 @@ impl BlockingIronOxide {
 
 /// Creates a tokio runtime with the default number of core threads (num of cores on a machine)
 fn create_runtime() -> tokio::runtime::Runtime {
-    tokio::runtime::Builder::new_multi_thread()
+    tokio::runtime::Builder::new_current_thread()
```

Comment:
Given that there's no upside to having multiple threads since all the calls use `block_on`, this is a good change. You should probably update the comment above.

Reply:
:+1: